src/lib.rs

```diff
@@ -63,6 +63,59 @@
 //! * `attiny85`
 //! * `attiny861`
 //! * `attiny88`
+//!
+//! # How to use this crate?
```

Comment:
Maybe mention that this is the low-level peripheral access crate (PAC) and there are higher level crates like `avr-hal` which implement a much easier to use interface ontop.